src/main/frontend/src/components/landingpage/InfiniteScrollComponent.js

```diff
@@ -0,0 +1,74 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import InfiniteScroll from "react-infinite-scroll-component";
+import { useEffect, useState } from "react";
+import LandingPageItem from "./LandingPageItem";
+import { Row } from "react-bootstrap";
+import {itemsFetch, itemsFetchByDate} from './itemService';
+
+function InfiniteScrollComponent(props) {
+  const [items, setItems] = useState([]);
+
+  const [hasMoreItems, setHasMoreItems] = useState(true);
+
+  const [page, setpage] = useState(0);
+
+  useEffect(async () => {
+    let data = "";
+
+    if(props.criteria === "normal"){
```

Comment:
lets create separate file called constants.js and inside have const LANDING_PAGE_TAB_VALUES: {
   NORMAL: "NORMAL",
   LAST_CHANCE: "LAST_CHANCE"
} and provide this values as prop, instead of plain text.